Roomsコントローラにindexアクションを追加する

uchihiro04 opened the pull request:
## 概要 
Roomコントローラの作成し、その中にindexアクションを追加しました。
現時点ではビューファイルに見出しのみ表示している状態です。

## スクリーンショット
* お部屋の内見一覧ページのPath
<img width="800" alt="image" src="https://github.com/uchihiro04/house-viewing-score-log/assets/77523896/3961ccfc-e81a-4d56-9aa1-e2a2d5846876">

* お部屋の内見一覧ページ
<img width="800" alt="image" src="https://github.com/uchihiro04/house-viewing-score-log/assets/77523896/706bb52f-171d-4833-8d8f-826a3a539f4f">

## 関連Issue
- #99 

## 参考にしたリンク
[Rails のルーティング \- Railsガイド](https://railsguides.jp/routing.html#%E3%83%8D%E3%82%B9%E3%83%88%E3%81%97%E3%81%9F%E3%83%AA%E3%82%BD%E3%83%BC%E3%82%B9)
[Rails 4\.2 URLにid以外のカラム値を使う \- カタカタブログ](https://totech.hateblo.jp/entry/2015/12/19/205018)


close #99

uchihiro04 committed:
Roomsコントローラの作成

uchihiro04 committed:
ルーティングの設定

内見一覧ページのPathである、/house_viewings/:house_viewing_uuid/roomsを設定しました。

uchihiro04 committed:
お部屋の内見一覧ページのビューファイルを作成

お部屋の内見一覧ページにあたるビューファイルを作成しました。

Yuma-Nagaoka commented on app/views/rooms/index.html.slim:
```diff
@@ -0,0 +1,2 @@
+h1 HouseViewings::Rooms#index
```
ここはHouseViewings::のネームスペースがついてますが自動生成されたものですか？
正直後で消しちゃうんであまり関係ないと思いますが、、ページの名前にしてしまいましょうか

uchihiro04 replied:
「HouseViewings::」に関しては、自動生成されたファイルを移動した際、見出しを変えていなかった形になります。

以下の修正を行いました。
* コントローラをHouseViewings::Roomsコントローラに変更
* コントローラのファイルの移動
* ビューファイルの移動
* Indexに関するビューファイル（app/views/house_viewings/rooms/index.html.slim）に見出し（内見するお部屋一覧）を設定

該当するコミットは[ca24925](https://github.com/uchihiro04/house-viewing-score-log/pull/104/commits/ca249250ff91a8438fb97fcb97416ceb63699730)、[7c18bea](https://github.com/uchihiro04/house-viewing-score-log/pull/104/commits/7c18beaaf10d70e781420acb7d6821be1e74d103)となります。

Yuma-Nagaoka replied:
これってRoomのモデルの方もHouseViewings::Roomにした方がいいみたいな話は久保木さんとの間で特になかったですか？

uchihiro04 replied:
Roomsのモデルに関してはハドルの中で話題には上がらなかったです。
モデルを名前空間にするかどうかは考えていなかったですね……。

uchihiro04 committed:
RoomsコントローラをHouseViewings::Roomsコントローラに変更

アプリの意図に沿った形になるように、HouseViewings::Roomsコントローラに変更を行いました。また、コントローラに関するファイルを移動しました。

uchihiro04 committed:
ビューファイルに見出しを追加・場所変更

ビューファイルに「内見したお部屋一覧」という見出しをつけました。また、コントローラの変更に伴いビューファイルの場所を変更しました。」

takuminasu reviewed (approved):
LGTM
